sys/arm/broadcom/bcm2835/bcm2838_pci.c

```diff
@@ -719,7 +723,18 @@ bcm_pcib_attach(device_t dev)
 	bcm_pcib_set_reg(sc, PCI_ID_VAL3,
 	    PCIC_BRIDGE << CLASS_SHIFT | PCIS_BRIDGE_PCI << SUBCLASS_SHIFT);
 
-	bcm_pcib_set_reg(sc, REG_BRIDGE_SERDES_MODE, 0x2);
+	uint32_t tmp = bcm_pcib_read_reg(sc, REG_PCIE_HARD_DEBUG);
```

Comment:
`tmp` should be declared at the top of this block with other declarations for style(9) compliance